### PR TITLE
polish (pl) translation - initial version

### PR DESCRIPTION
**Pull request by janslu**
## What does this PR do?

It adds polish translation to talk 
## How do I test this PR?
npm run test throws many errors but none seems to be caused by adding translations... I have looked into a Finnish translation and corrected the same test file... Hope everything is fine with this commit.

### CODE REVIEW

**Comment by kgardnr** [comment]
Thank you @janslu!

**Review by kgardnr — APPROVED** [review]

File: src/locales/pl/account.ftl [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+download-landingPage-downloadComments =
+  Download My Comment History
+download-landingPage-sorry = Your download link is invalid.
+
+## Unsubscribe
+
+unsubscribe-oopsSorry = Oops Sorry!
+unsubscribe-successfullyUnsubscribed = You are now unsubscribed from all notifications
+unsubscribe-clickToConfirm = Click below to confirm that you want to unsubscribe from all notifications.
+unsubscribe-confirm = Confirm
```
Comment (wyattjoh) [inline src/locales/pl/account.ftl:61]: Missed translations

Comment (wyattjoh) [reply src/locales/pl/account.ftl:61]: Tracking fix in CORL-936

Comment (janslu) [reply src/locales/pl/account.ftl:61]: Created updated pull request (#2852) with missing phrases and updated numerals. Sorry and thank you for noticing.

**Pull request merged**